sdk/eventgrid/azure-messaging-eventgrid-cloudnative-cloudevents/README.md

```diff
@@ -0,0 +1,147 @@
+# Azure Cloud Native Cloud Event support for Event Grid
+
+This library can be used to enable publishing the Cloud Native Computing Foundation(CNCF) [CloudEvents][CNCFCloudEvents]
+using the Azure Event Grid library. 
+
+## Getting started
+
+### Prerequisites
+You should have an EventGrid client before using this bridge library. Follow [Azure EventGrid][eventgridGettingStarted]
+steps to create an EventGrid client.
+
+### Include the package
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on GA version of the library. In the following snippet, replace the {bom_version_to_target} placeholder with the version number.
+To learn more about the BOM, see the [AZURE SDK BOM README](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/boms/azure-sdk-bom/README.md).
+
+```xml
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-sdk-bom</artifactId>
+            <version>{bom_version_to_target}</version>
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>
+    </dependencies>
+</dependencyManagement>
+```
+and then include the direct dependency in the dependencies section without the version tag.
+
+```xml
+<dependencies>
+  <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-messaging-eventgrid-cloudnative-cloudevents</artifactId>
+      <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-messaging-eventgrid-cloudnative-cloudevents;current} -->
+  </dependency>
+</dependencies>
+```
```

Comment:
We should skip this section until we have a GA version as  BOM only includes GA'd libraries.